lib/datasource/helm/index.js

```diff
@@ -0,0 +1,84 @@
+const yaml = require('js-yaml');
+const got = require('../../util/got');
+
+module.exports = {
+  getPkgReleases,
+};
+
+async function getPkgReleases({ lookupName, helmRepository }) {
+  if (!lookupName) {
+    logger.warn(`lookupName was not provided to getPkgReleases`);
+    return null;
+  }
+  if (!helmRepository) {
+    logger.warn(`helmRepository was not provided to getPkgReleases`);
+    return null;
+  }
+  const cacheNamespace = 'datasource-helm';
+  const cacheKey = helmRepository;
+  const cachedIndex = await renovateCache.get(cacheNamespace, cacheKey);
+  let res;
+  if (!cachedIndex) {
+    try {
+      res = await got('index.yaml', { baseUrl: helmRepository });
+      if (!res || !res.body) {
+        logger.warn(
+          { dependency: lookupName },
+          `Received invalid index.yaml from ${helmRepository}`
```

Comment:
Technically we haven't tried parsing the yaml yet so this message might confuse people.